crates/archlint/test_data/abstractness/useless/unused.ts

```
@@ -1,2 +1,2 @@
-export type Abstract = { a: number };
-import { something } from './somewhere';
+export interface IUnused {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)